package.json

```diff
@@ -21,6 +21,7 @@
     "sirv-cli": "^1.0.6",
     "sucrase": "^3.15.0",
```

Comment:
you can probably remove this now

Reply:
This is used to run the tests via `uvu` and predates the change of the main compilation to `sucrase` from `typescript`. I don't know if it's better to swap this out for `ts-node` but I'm not the right person to make that decision so it might be better left for a future PR.